lib/bloc/refund/refund_state.dart

```diff
@@ -3,20 +3,29 @@ import 'package:breez_sdk/bridge_generated.dart';
 class RefundState {
   final List<SwapInfo>? refundables;
   final String? refundTxId;
-  final String? error;
+  final String? refundablesError;
+  final String? refundError;
 
-  RefundState({this.refundables, this.refundTxId, this.error = ""});
+  RefundState({
+    this.refundables,
+    this.refundTxId,
+    this.refundablesError = "",
+    this.refundError = "",
+  });
 
   RefundState.initial() : this();
 
   RefundState copyWith({
     List<SwapInfo>? refundables,
     String? refundTxId,
-    String? error,
+    String? refundablesError,
+    String? refundError,
   }) {
     return RefundState(
-        refundables: refundables ?? this.refundables,
-        refundTxId: refundTxId ?? this.refundTxId,
-        error: error ?? this.error);
+      refundables: refundables ?? this.refundables,
+      refundTxId: refundTxId ?? this.refundTxId,
+      refundablesError: refundablesError ?? this.refundablesError,
```

Comment:
Not sure we need the refundablesError as this indicates a bug in the sdk.
Perhaps just logging this error is enough.

Reply:
Done.
- Remove refundablesError 76badf4